test/e2e/storage/testpatterns/testpattern.go

```diff
@@ -55,12 +56,14 @@ var (
 
 // TestPattern represents a combination of parameters to be tested in a TestSuite
 type TestPattern struct {
-	Name         string                  // Name of TestPattern
-	FeatureTag   string                  // featureTag for the TestSuite
-	VolType      TestVolType             // Volume type of the volume
-	FsType       string                  // Fstype of the volume
-	VolMode      v1.PersistentVolumeMode // PersistentVolumeMode of the volume
-	SnapshotType TestSnapshotType        // Snapshot type of the snapshot
+	Name           string                      // Name of TestPattern
+	FeatureTag     string                      // featureTag for the TestSuite
+	VolType        TestVolType                 // Volume type of the volume
+	FsType         string                      // Fstype of the volume
+	VolMode        v1.PersistentVolumeMode     // PersistentVolumeMode of the volume
+	SnapshotType   TestSnapshotType            // Snapshot type of the snapshot
+	BindingMode    storagev1.VolumeBindingMode // VolumeBindingMode of the volume
```

Comment:
Is this used anywhere?

Reply:
Let's open up a separate issue to discuss this further but I want to move away from being able to toggle/customize StorageClass arguments in the framework tests.  Now that these tests are also used externally by CSI driver developers, I see a use case where a vendor wants to test their plugin with specific recommended/default storage classes that they provide.

And in some cases, toggling these parameters automatically in the test suite may not work for the storage plugin.  For example, immediate binding mode is not going to work in a multi-zone cluster.  It would be better in this case if the vendor can completely control all the parameters instead of the test suite.